NitroxLauncher/Pages/ServerPage.xaml.cs

```diff
@@ -1,37 +1,850 @@
+
+                UpdateVisualWorldSettings();
+
+                ImportSaveBtnBorder.Opacity = 0;
+                ImportSaveBtn.IsEnabled = false;
+                ImportWorldBox.Opacity = 0;
+                ImportWorldBox.IsHitTestVisible = false;
+                TBWorldSeed.IsReadOnly = true;
+
+                LauncherNotifier.Success("Successfully imported the selected save file");
+            }
+            catch (Exception ex)
+            {
+                LauncherNotifier.Error("Failed to import the selected save file. Please check your log for details.");
+                Log.Error($"Could not import save \"{Path.GetFileName(SelectedWorldDirectory)}\" : {ex.GetType()} {ex.Message}");
+            }
+        }
+
+        private void ImportWorldCancelBtn_Click(object sender, RoutedEventArgs e)
+        {
+            ImportWorldBox.Opacity = 0;
+            ImportWorldBox.IsHitTestVisible = false;
+        }
+
+        // Start Server button
+        private void StartServer_Click(object sender, RoutedEventArgs e)
+        {
+            SelectedWorldDirectory = WorldManager.GetSaves().ElementAtOrDefault(WorldListingContainer.SelectedIndex)?.WorldSaveDir ?? "";
+
+            if (!WorldManager.GetSaves().ElementAtOrDefault(WorldListingContainer.SelectedIndex).IsValidSave)
+            {
+                LauncherNotifier.Error($"This save is an invalid version.");
+                return;
+            }
+            else if (!Directory.Exists(WorldManager.GetSaves().ElementAtOrDefault(WorldListingContainer.SelectedIndex)?.WorldSaveDir))
+            {
+                LauncherNotifier.Error($"This save does not exist or is not valid.");
+                InitializeWorldListing();
+                return;
+            }
+
+            try
+            {
+                LauncherLogic.Server.StartServer(RBIsExternal.IsChecked == true, SelectedWorldDirectory);
+            }
+            catch (Exception ex)
+            {
+                if (ex.ToString().Contains("An instance of Nitrox Server is already running"))
+                {
+                    LauncherNotifier.Error("An instance of the Nitrox server is already running, please close it to start another server.");
+                }
+                else
+                {
+                    MessageBox.Show(ex.ToString(), "Error", MessageBoxButton.OK, MessageBoxImage.Error);
+                }
+                return;
+            }
+
+            if (File.Exists(Path.Combine(SelectedWorldDirectory, "WorldData.json")))
+            {
+                File.SetLastWriteTime(Path.Combine(SelectedWorldDirectory, "WorldData.json"), DateTime.Now);
+            }
+            
+            WorldDirCurrentlyUsed = SelectedWorldDirectory;
+            InitializeWorldListing();
+        }
+
+    }
+
+    // OPTIONAL - Only used to view world listings in intellisense, in addition to the lines that are in the ListView definition in ServerPage.xaml
+    public class World_Listing
```

Comment:
Class name does not match project code style.

```suggestion
    public class WorldListing
```